src/pretix/base/settings.py

```diff
@@ -2442,10 +2442,29 @@ def unserialize(cls, s):
 ])
 
 PERSON_NAME_SALUTATIONS = [
-    pgettext_lazy("person_name_salutation", "Ms"),
-    pgettext_lazy("person_name_salutation", "Mr"),
+    "Ms",
+    "Mr",
+    "Mx",
```

Comment:
`makemessages` is statically analyzing the code to find all gettext_usages, but it's not clever enough to evaluate the dynamic call you introduced in `questions.py`. Therefore, "Mr" will never show up in the .po file, and therefore our translators will never see or translate it.

We'll need to have the static pgettext-statment *somewhere* in our codebase. Usually, ``gettext_noop`` would solve that, but there unfortunately is no ``pgettext_noop``. To avoid changing the makemessages command in Django, the easiest solution would probably be to put this like a choice array:

```
PERSON_NAME_SALUTATIONS = [
    ("Ms", pgettext_lazy("person_name_salutation", "Ms")),
    ("Mr", pgettext_lazy("person_name_salutation", "Mr")),
    ("Mx", pgettext_lazy("person_name_salutation", "Mx")),
]
```

and then use it appropriately in the form

Reply:
Good idea, I changed it accordingly.